python-threatexchange/threatexchange/api_models.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+from datetime import datetime
+from dateutil.parser import parse
+from dataclasses import dataclass
+
+"""
+Models (dataclasses only) for interfacing with the threatexchange API.
```

Comment:
(ignroable): I have never seen a specific section of libraries split out and called "models" before, and generally find the convention confusing. If you had a models.py class in a directory and asked me what I expected to be in there, I would guess simulation or ML before I would guess MVC.

This is probably due to me never explicitly building something with strict MVC in mind, and it's unclear to me still what following the conventions get me.

Reply:
These are DTO objects for interfacing with the Graph APIs. You are correct, they are not models. I'll figure out a better name. I've used entities.py, representations.py etc before. Perhaps shapes.py?

Reply:
Besides Hack, do other languages use something known specifically as a `shape`? I think we should stick to a name that's a bit more generic. It seems very natural to me to have something I might call a domain model object like `ThreatPrivacyGroup` in a module with models in the name. Whether it's an MVC model or a domain model, these seem to be models to me. Either that or entities. I get it that there's a very ML-specific thing called a model, too, but given all our other code (and the lack of anything ML-ish) I'm not thinking there's really any confusion here?